clusterman/kubernetes/util.py

```diff
@@ -60,7 +60,18 @@ def __init__(self, kubeconfig_path: str, client_class: Type) -> None:
         :param Type client_class: k8s client class to initialize
         """
         try:
-            kubernetes.config.load_kube_config(kubeconfig_path, context=os.getenv("KUBECONTEXT"))
+            """
+            https://kubernetes.io/docs/concepts/containers/container-environment/#container-environment
+            Every pod in k8s get some default enviornment variable injected, Including KUBERNETES_SERVICE_HOST
+            which points to K8s service. We are using this variable to distinguise between when cluterman is started in
```

Comment:
it points to the host that the pod is running in, no? not to a k8s Service?

Reply:
It points to default k8s service, which is used to connect to API server.

Reply:
ah gotcha, might be worth rewording this a bit to "points to the default `kubernetes` service" since it's a bit ambiguous right now